pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -338,6 +338,16 @@ func (builder *podEntityDTOBuilder) getPodCommoditiesSold(
 	}
 	commoditiesSold = append(commoditiesSold, podAccessComm)
 
+	// label commodity
+	labelComm, err := sdkbuilder.NewCommodityDTOBuilder(proto.CommodityDTO_LABEL).
+		Key(string(pod.UID)).
+		Capacity(labelCommodityDefaultCapacity).
+		Create()
+	if err != nil {
+		return nil, err
+	}
+	commoditiesSold = append(commoditiesSold, labelComm)
+
```

Comment:
We should not create `LABEL`  sold commodities for pods. We create `VMPM_ACCESS` commodities between pod sold and container bought to bind the container to the pod, such that it cannot be moved out of the pod in Market.